lib/index.ts

```diff
@@ -1,25 +1,39 @@
-import { useState } from 'react';
+import { useCallback, useState } from 'react';
 import { unstable_batchedUpdates as batch } from 'react-dom';
 
 type UpdateRecord = [(value: any) => void, unknown];
 type UpdateRecordList = UpdateRecord[];
 
-let updates: UpdateRecordList = [];
+const updates: { list: UpdateRecordList } = { list: [] };
 let updatePromise: Promise<void> | null = null;
 
+function pushUpdate(updateRecord: UpdateRecord): void {
+  updates.list.push(updateRecord);
+}
+
+function flushUpdates(): void {
+  updates.list = [];
+}
+
+function getUpdates() {
+  return updates.list;
+}
+
```

Comment:
Why is there an object for `updates` and these new functions? I'm just wondering if it's for clean code / linting.

Reply:
We need some global store for updates accessible from useCallback, which preserves its reference after we clean the store. For this I wrote functions that access the store via a reference. I could've used something like ```arr.length = 0```  to avoid creation of separate functions, but according to this [benchmark](https://jsben.ch/hyj65) it's slightly slower than just assigning empty array. I also didn't want to write a class, since for now the support includes ie11, and it would result in some additional transpiled code. Feel free to suggest better approach.

Reply:
Why is there an array for `updates`? Why can't `updates` just be `[]`, like it was before?

Reply:
Ah, my bad. Now I see, what you are talking about. Added to pr.